docs/devsite-help/troubleshooting.md

```diff
@@ -218,3 +218,62 @@ native libraries are copied appropriately.
 
 This information is now in the [protocol buffers guide](protobuf.md#repeated-fields-and-maps); this section has been
 retained for the sake of existing links.
+
+## How can I get ErrorInfo from RpcException?
+
+`ErrorInfo` in `RpcException` provides a consistent and human readable error details.
+It contains three major fields - reason, domain and metadata described as below:
+
+- `Reason` - It is a constant value and indicates the proximate cause of 
+the error. Error reasons are unique within a particular domain of errors.
+
+- `Domain` - It is the logical grouping to which the error belongs. It is 
+typically the registered service name of the tool or product that 
+generates the error. Example: "pubsub.googleapis.com". If the error 
+is generated by some common infrastructure, the error domain must be 
+a globally unique value that identifies the infrastructure. 
+For Google API infrastructure, the error domain is "googleapis.com".
+
+- `Metadata`- It is the additional structured details about the error. 
+It can help in identifying the exact cause of the error.
```

Comment:
You need to specify the way in which this is constant. (I understand this text is taken from error_details.proto, but we can clarify here.)

I'd also avoid the "It is" wording here and below. Here's a potential rewriting:

```suggestion
- `Reason`: the proximate cause of the error. This is a stable unique identifier for this
   type of error within the domain, and can be used to programmatically determine
   how to handle the error.
- `Domain`: the logical grouping to which the error belongs. The error domain
   is typically the registered service name of the tool or product that
   generates the error. Example: "pubsub.googleapis.com". If the error 
   is generated by some common infrastructure, the error domain will be 
   a globally unique value that identifies the infrastructure. 
For Google API infrastructure, the error domain is "googleapis.com".
- `Metadata`: additional structured details about the error.
```

Reply:
Incorporated changes.